### PR TITLE
[16.0][IMP] sale_partner_selectable_option

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
This PR improves the process of choosing the default value for sale_selectable by utilizing a system parameter instead of hardcoding it.

@qrtl

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
LGTM

**Review by victoralmau — APPROVED** [review]

File: sale_partner_selectable_option/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -1 +1,3 @@
 This module limits the partners to be selected in the sales orders according to the "Selectable in orders" field.
+By default, this field is set to True, allowing all partners to be selectable. The default setting can be modified
+by adjusting the system parameter with the key 'sale_partner_selectable_option.default_sale_selectable'.
```
Comment (yostashiro) [inline sale_partner_selectable_option/readme/DESCRIPTION.rst]: @AungKoKoLin1997 Please see https://github.com/OCA/purchase-workflow/pull/2157#discussion_r1464705080.

**Review by yostashiro — APPROVED** [review]
LGTM.

**Comment by AungKoKoLin1997** [comment]
@rousseldenis Could we please have this PR merged?

**Pull request merged**